sreejasuresh007-P1.md

```diff
@@ -0,0 +1,182 @@
+# Loading data from csv file into postgresql using python
+
+For any kind of applications, one of the major requirements is to store, modify and access data for viewing, analysing and finding insights from it. We are using PostgreSQL as it is an open source, highly stable and can be easily integrated with cloud too. We use **psycopg2** module of python for the connection with database. 
+
+Sample data we will be using for this example:
+
+stu_id | stu_firstname | stu_lastname | department
+------------ | -------------  | ------------- | -------------
+189056 | John | Dodge | CSE
+189057 | Sarah | Smith | ECE
+189058 | George | Dapper | EEE
+189059 | Jack | Dodge | ME
+189060 | Jessica | Jones | CSE
+189061 | Dragon | Davich | ECE
+189062 | Steve | Curio | EEE
+189063 | aura | Black | IT
+189064 | Wilhelm | Blake | IT
+
+**connect** method is used for establishing connection with database. The parameters are: 
+
+**‘user’**: the owner of database; 
+
+**‘password’**: password to access the database;
+
+**‘host’**: host where Postgres server is currently running; localhost if local server is used, else hostname of remote server;
+
+**‘port’**: port number of database;
+
+**‘database’**: database name in which data has to be stored;
+
+1. Username is important as there are multiple users who are given different kinds of access to the database. This is done to increase security levels of data. Similarly multiple databases can be used to increase data security and division. 
+
+```
```

Comment:
Add the language tag for all the code blocks for proper syntax highlighting like [this](https://github.com/blrgis/blrgis.github.io/pull/1/files#diff-ba4b79cd400f294effc0b7536c5c3ee2R13)

Reply:
Done